<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lab1Class</class>
 <widget class="QWidget" name="Lab1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1068</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab1</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>695</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2,2">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Formation</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QRadioButton" name="centerButton">
              <property name="text">
               <string>Centered</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="randomButton">
              <property name="text">
               <string>Random</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="circleButton">
              <property name="text">
               <string>Circle</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Elements
Number</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSlider" name="Number">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>25</number>
              </property>
              <property name="value">
               <number>25</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Placing</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="2" column="1">
             <widget class="QSlider" name="Size">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="value">
               <number>99</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Element
Size</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Element
Angle</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QSlider" name="ElementAngle">
              <property name="maximum">
               <number>360</number>
              </property>
              <property name="value">
               <number>359</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Scene
Angle</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QSlider" name="SceneAngle">
              <property name="maximum">
               <number>360</number>
              </property>
              <property name="value">
               <number>359</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Lines
Width</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSlider" name="Width">
              <property name="minimum">
               <number>50</number>
              </property>
              <property name="maximum">
               <number>500</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Colors</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Background</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>R</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSlider" name="BR">
                 <property name="maximum">
                  <number>255</number>
                 </property>
                 <property name="value">
                  <number>255</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>G</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>B</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QSlider" name="BG">
                 <property name="maximum">
                  <number>255</number>
                 </property>
                 <property name="value">
                  <number>255</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QSlider" name="BB">
                 <property name="maximum">
                  <number>255</number>
                 </property>
                 <property name="value">
                  <number>255</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Foreground</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>R</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSlider" name="FR">
                 <property name="maximum">
                  <number>255</number>
                 </property>
                 <property name="value">
                  <number>255</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>B</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QSlider" name="FG">
                 <property name="maximum">
                  <number>255</number>
                 </property>
                 <property name="value">
                  <number>255</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>G</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QSlider" name="FB">
                 <property name="maximum">
                  <number>255</number>
                 </property>
                 <property name="value">
                  <number>255</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>A</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QSlider" name="FA">
                 <property name="maximum">
                  <number>255</number>
                 </property>
                 <property name="value">
                  <number>255</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Scene
scaling</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="Scale">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>99</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Lab1.qrc"/>
 </resources>
 <connections/>
</ui>
